uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_description/config/costmap_parameters.yaml

```diff
@@ -0,0 +1,217 @@
+controller_server:
```

Comment:
I assume this is just the default file given from the nav2 guide, right?

Reply:
Also, how did you tweak the params? It's fine if these are just "arbitrary" for now... but we just need to know if we need to come back to them if something isnt working

Reply:
All the controller_server params are just from the guide, and the params are not tweaked